db/migrate/20230825233612_add_author_id_to_posts.rb

```diff
@@ -0,0 +1,8 @@
+class AddAuthorIdToPosts < ActiveRecord::Migration[7.0]
+  def change
+    Comment.delete_all
+    Post.delete_all
+    add_column :posts, :author_id, :integer, null: false
+  end
+end
```

Comment:
- [ ] This migration file is causing your migration not to run, because author_id already exist in create post-migration, consider removing it